src/Uno.UI/UI/Xaml/FrameworkElement.Interface.netstdref.cs

```diff
@@ -48,7 +48,22 @@ private protected virtual void OnUnloaded()
 		{
 		}
 
-		public TransitionCollection Transitions { get => throw new NotImplementedException(); set => throw new NotImplementedException(); }
+		#region Transitions Dependency Property
+
+		[GeneratedDependencyProperty(DefaultValue = null, ChangedCallback = true)]
+		public static DependencyProperty TransitionsProperty { get; } = CreateTransitionsProperty();
+
+		public TransitionCollection Transitions
+		{
+			get => GetTransitionsValue();
+			set => SetTransitionsValue(value);
+		}
+
+		private void OnTransitionsChanged(DependencyPropertyChangedEventArgs args)
```

Comment:
Same here, empty callback

Reply:
This one is not used (reference implementation)